source/sass/wagtail/blocks.scss

```diff
@@ -2,12 +2,20 @@
   @import "./profiles";
 }
 
-.wide {
+.full-width {
```

Comment:
renaming wide to full-width, as this is the class that allows us to have the end to end content